src/models/menu.ts

```diff
@@ -73,13 +73,13 @@ const getBreadcrumbNameMap = menuData => {
   const routerMap = {}
 
   const flattenMenuData = data => {
```

Comment:
please could you use function keyword here, I think it will increase the readability.

```
function flattenMenuData (data) ...
```

Reply:
modified this function and its parent.